runtime/dart_isolate.cc

```diff
@@ -251,23 +251,43 @@ std::weak_ptr<DartIsolate> DartIsolate::CreateRootIsolate(
   DartErrorString error;
   Dart_Isolate vm_isolate = nullptr;
   auto isolate_flags = flags.Get();
-  /// TODO(b/72025) This will be where we call Dart_CreateIsolateInGroup if
-  /// spawning_isolate != nullptr.
-  vm_isolate = CreateDartIsolateGroup(
-      std::move(isolate_group_data), std::move(isolate_data), &isolate_flags,
-      error.error(),
-      [](std::shared_ptr<DartIsolateGroupData>* isolate_group_data,
-         std::shared_ptr<DartIsolate>* isolate_data, Dart_IsolateFlags* flags,
-         char** error) {
-        return Dart_CreateIsolateGroup(
-            (*isolate_group_data)->GetAdvisoryScriptURI().c_str(),
-            (*isolate_group_data)->GetAdvisoryScriptEntrypoint().c_str(),
-            (*isolate_group_data)->GetIsolateSnapshot()->GetDataMapping(),
-            (*isolate_group_data)
-                ->GetIsolateSnapshot()
-                ->GetInstructionsMapping(),
-            flags, isolate_group_data, isolate_data, error);
-      });
+
+  IsolateMaker isolate_maker;
+#if FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_PROFILE || \
+    FLUTTER_RUNTIME_MODE == FLUTTER_RUNTIME_MODE_RELEASE
+  if (spawning_isolate) {
+    isolate_maker =  [spawning_isolate](
+            std::shared_ptr<DartIsolateGroupData>* isolate_group_data,
+            std::shared_ptr<DartIsolate>* isolate_data,
+            Dart_IsolateFlags* flags, char** error) {
+      return Dart_CreateIsolateInGroup(
+          /*group_member=*/spawning_isolate->isolate(),
+          /*name=*/"spawn",
```

Comment:
What's the impact of this name? Add some comments?

Reply:
I'm not sure, I moved over the same name selection from the other section.